eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -4,7 +4,7 @@ parameters:
   WorkingDirectory: '$(System.DefaultWorkingDirectory)'
   ScriptDirectory: 'eng/common/scripts'
   Recursive: $false
-  CheckLinkGuidance: $false
+  CheckLinkGuidance: $true
```

Comment:
Lets not modify this eng/common file but instead explicitly pass true where we use it in the java repo.